st3/lsp_utils/npm_client_handler_v2.py

```diff
@@ -100,7 +100,11 @@ def configuration(cls) -> Tuple[sublime.Settings, str]:
     def _upgrade_languages_list(cls, languages):
         upgraded_list = []
         for language in languages:
-            if 'scopes' in language:
+            if 'document_selector' in language:
+                language.pop('scopes', None)
+                language.pop('syntaxes', None)
+                upgraded_list.append(language)
+            elif 'scopes' in language:
                 upgraded_list.append({
                     'languageId': language.get('languageId'),
                     'document_selector': ' | '.join(language.get('scopes')),
```

Comment:
Like I mentioned in the previous PR that introduced this, using `scopes` for the `document_selector` is wrong and you should use `syntaxes` instead. I think this introduced the bug. Anyway, this probably avoids the problem / solves it in a different way :)

Reply:
You weren't strict enough in stopping me from integrating it. :)